owlbot.py

```diff
@@ -22,8 +22,15 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
+excludes = [
+    "noxfile.py",  # pytype
+    "setup.cfg",  # pytype
+    ".flake8",  # flake8-import-order, layout
+    ".coveragerc",  # layout
+    "CONTRIBUTING.rst",  # no systests
```

Comment:
Can we keep `CONTRIBUTING.rst` and just update the SHA in `.OwlBot.lock.yaml` to `b6169fc6a5207b11800a7c002d0c5c2bc6d82697185ca12e666f44031468cfcd`?

Reply:
There should be no "System Tests" section at all in `CONTRIBUTING.rst` for this repository -- it has none.  I don't know that the generator is configurable enough to allow that change, and I'd really like not to have to add the monster `s.replace` required to remove it.

Reply:
Makes sense.